src/components/chat/chat-panel.tsx

```diff
@@ -22,8 +23,14 @@ interface IProps {
   onCloseChatPanel:(show:boolean) => void;
 }
 
+let storedDocument: IDocumentMetadata | ICurriculumMetadata;
+
 export const ChatPanel: React.FC<IProps> = ({ user, activeNavTab, focusDocument, focusTileId, onCloseChatPanel }) => {
+
   const document = useDocumentOrCurriculumMetadata(focusDocument);
+  if ( isCurriculumMetadata(document)){ //we always pass storedDocument (never undefined) into <CommentedDocuments>
+    storedDocument = document;
+  }
```

Comment:
True, lets talk about this in our zoom meeting, I actually forgot to include this in the line above. 

  const document = useDocumentOrCurriculumMetadata(focusDocument || "sas/0/1/introduction");
  
  therefore when focusDocument is undefined, it'll default to a valid section in the clue document, however this isn't universal for all curriculum. 
  
 But in the example of CLUE 0.1 - when focusDocument is undefined at the very beginning, it would default to /sas/0/1/introduction which therefore document would be a curriculum document and storedDocument would never be undefined.